src/converters/utils/calculations/checkWeekend.ts

```
@@ -0,0 +1,8 @@
+import { getDay } from "date-fns";
+import { getMultipleValuesInput } from "../../../common/utils";
+
+export const checkWeekend = (date: Date | number) => {
+    const currentDay = getDay(date);
+    const weekends = getMultipleValuesInput('WEEKENDS').map((el) => parseInt(el));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)